manifests/init.pp

```diff
@@ -26,33 +26,33 @@
 # }
 class nginx (
   ### START Nginx Configuration ###
-  $client_body_temp_path                                     = $::nginx::params::client_body_temp_path,
-  Boolean $confd_only                                        = false,
-  Boolean $confd_purge                                       = false,
-  $conf_dir                                                  = $::nginx::params::conf_dir,
+  Stdlib::Absolutepath $client_body_temp_path                = undef,
```

Comment:
`undef` isn't a valid default for the data type (and the default is overridden in the module data anyway).